dataprofiler/__init__.py

```diff
@@ -22,3 +23,10 @@
         '\tsudo apt-get -y install libsnappy-dev`\n',
         ImportWarning
     )
+
+def set_seed(seed=None):
+    # also check it's an integer
+    if seed is not None and (seed < 0 or int(seed)!=float(seed)):
```

Comment:
if seed were a `{}` it would not have a `<` operator. hence i think we need to alter this to check specifically that it is an `int`
e.g. `if seed is not None and (not isinstance(seed, int) or seed < 0):`